crates/evm/src/trace/mod.rs

```diff
@@ -2,11 +2,8 @@ use crate::{
     abi::CHEATCODE_ADDRESS, debug::Instruction, trace::identifier::LocalTraceIdentifier, CallKind,
 };
 pub use decoder::{CallTraceDecoder, CallTraceDecoderBuilder};
-use ethers::{
-    abi::{ethereum_types::BigEndianHash, Address, RawLog},
-    core::utils::to_checksum,
-    types::{Bytes, DefaultFrame, GethDebugTracingOptions, StructLog, H256, U256},
-};
+use ethers::types::{DefaultFrame, GethDebugTracingOptions, StructLog};
```

Comment:
These types should also go to alloy? I guess RPC crate under core/?

Think good opportunity to get that started, along with the TxRequest/Response/Receipt?